treeherder/webapp/api/performance_serializers.py

```diff
@@ -653,6 +654,208 @@ class Meta:
+    base_kde = KDESerializer(many=False)
+    new_kde = KDESerializer(many=False)
+
+
+class PerfCompareResultsSerializerV2(serializers.ModelSerializer):
+    base_rev = serializers.CharField()
+    new_rev = serializers.CharField()
+    base_app = serializers.CharField(
+        max_length=10,
+        default="",
+    )
+    new_app = serializers.CharField(
+        max_length=10,
+        default="",
+    )
+    is_complete = serializers.BooleanField()
+    platform = serializers.CharField()
+    header_name = serializers.CharField()
+    base_repository_name = serializers.CharField()
+    new_repository_name = serializers.CharField()
+    base_measurement_unit = serializers.CharField(default="")
+    new_measurement_unit = serializers.CharField(default="")
+    base_retriggerable_job_ids = serializers.ListField(child=serializers.IntegerField(), default=[])
+    new_retriggerable_job_ids = serializers.ListField(child=serializers.IntegerField(), default=[])
+    option_name = serializers.CharField()
+    base_runs = serializers.ListField(
+        child=PerfCompareDecimalField(),
+        default=[],
+    )
+    new_runs = serializers.ListField(
+        child=PerfCompareDecimalField(),
+        default=[],
+    )
+    base_runs_replicates = serializers.ListField(
+        child=PerfCompareDecimalField(),
+        default=[],
+    )
+    new_runs_replicates = serializers.ListField(
+        child=PerfCompareDecimalField(),
+        default=[],
+    )
+    base_mean = PerfCompareDecimalField()
+    new_mean = PerfCompareDecimalField()
+    base_median = PerfCompareDecimalField()
+    new_median = PerfCompareDecimalField()
+    base_stddev = PerfCompareDecimalField()
+    new_stddev = PerfCompareDecimalField()
+    base_count = serializers.IntegerField()
+    new_count = serializers.IntegerField()
+    base_variance = PerfCompareDecimalField()
+    new_variance = PerfCompareDecimalField()
+    base_stddev_pct = PerfCompareDecimalField()
+    new_stddev_pct = PerfCompareDecimalField()
+    delta_value = PerfCompareDecimalField()
+    delta_percentage = PerfCompareDecimalField()
+    new_is_better = OptionalBooleanField()
+    lower_is_better = OptionalBooleanField()
+    is_confident = OptionalBooleanField()
+    noise_metric = OptionalBooleanField(default=False)
+    graphs_link = serializers.CharField()
+    more_runs_are_needed = OptionalBooleanField(default=False)
+    is_fit_good = OptionalBooleanField(default=True)
+    is_improvement = serializers.BooleanField(required=False)
+    is_regression = serializers.BooleanField(required=False)
+    is_meaningful = serializers.BooleanField(required=False)
+    base_parent_signature = serializers.IntegerField()
+    new_parent_signature = serializers.IntegerField()
+    base_signature_id = serializers.IntegerField()
+    new_signature_id = serializers.IntegerField()
+    has_subtests = serializers.BooleanField()
+    ks_test = StatisticsTestSerializer(many=False)
```

Comment:
I like how you grouped multiple fields into different serializers! Do you think it's possible to do the same for cliff / cles ?